mozdownload/scraper.py

```diff
@@ -823,6 +845,7 @@ def get_build_info_for_index(self, build_index=None):
         parser = DirectoryParser(url, authentication=self.authentication,
                                  timeout=self.timeout_network)
         parser.entries = parser.filter(r'^\d+$')
+        parser.entries = parser.filter(self.is_build_dir)
```

Comment:
Hm, `is_build_dir` should return a boolean when I look at its name. Maybe we can rename it right away to something better? Especially when we want to also download other types as builds, the name would not match.

Feel free to file a new issue for that.